utils/sprint_general_labels.js

```diff
@@ -1,39 +1,10 @@
 import Baracoda from '@/utils/baracoda'
 import { query, headers } from '@/utils/sprint_constants'
+import layoutTemplate from '@/config/templates/sprint_general_label_layout.json'
+import TemplateRenderer from './template_renderer'
 
 const config = useRuntimeConfig()
-
-// Will create a new layout object for a print job
-// Requires barcode which will be used for barcode and text field
-// TODO: DPL-561 - how do we turn this into external json
```

Comment:
I wasn't 100% sure myself but I convinced myself that it was a worthy thing to do.  It is good to get the boilerplate structure of the body into a JSON file rather than littering the JavaScript files with it.  Especially since it's 98% static content and only a couple of insertion points.  Mustache is a well established templating engine so it could be useful in other areas as well.